docs/test_candlestick_agg.md

```diff
@@ -84,7 +81,7 @@ GROUP BY daily_bucket,symbol;
 ```
 
 ```output
-      daily_bucket      | symbol | vwap |  high
-------------------------+--------+------+---------
- 2023-01-11 00:00:00+00 | AAPL   | NULL | 133.445
+      daily_bucket      | symbol | vwap | high
```

Comment:
I'm not actually sure we need a rollup test here, since the cagg doesn't affect the rollup behavior at all, but if we want to have it we should look at a few more columns so we can make sure it's behaving like we expect.